src/js/utils/ApplicationUtil.js

```diff
@@ -0,0 +1,60 @@
+import {
+  MESOS_SUMMARY_CHANGE,
+  MESOS_SUMMARY_REQUEST_ERROR
+} from '../constants/EventTypes';
+import Config from '../config/Config';
+import MesosSummaryStore from '../stores/MesosSummaryStore';
+
+module.exports = {
+  /**
+   * Checks that enough time has elapsed before rendering app
+   *
+   * @param  {Func} onDelayEndCallback Gets called after delay has ellapsed
+   */
+  renderOnDelayEnd(onDelayEndCallback) {
```

Comment:
Just an idea but seeing this isn't actually doing the rendering, maybe it could be `invokeAfterPageLoad(fn, delay)` and you could pass in `Config.applicationRenderDelay` as the `delay` argument to keep this more functional.